libraries/chain/controller.cpp

```diff
@@ -4164,9 +4170,11 @@ struct controller_impl {
                   }
                }
             } else if (!branches.first.empty()) {
-               ilog("applying ${n} fork db blocks from ${cbn}:${cbid} to ${nbn}:${nbid}",
-                    ("n", branches.first.size())("cbid", (*branches.first.rbegin())->id())("cbn", (*branches.first.rbegin())->block_num())
-                    ("nbid", new_head->id())("nbn", new_head->block_num()));
+               if (fc::time_point::now() - new_head->timestamp() < fc::minutes(5)) {
+                  ilog("applying ${n} fork db blocks from ${cbn}:${cbid} to ${nbn}:${nbid}",
+                       ("n", branches.first.size())("cbid", (*branches.first.rbegin())->id())("cbn", (*branches.first.rbegin())->block_num())
+                       ("nbid", new_head->id())("nbn", new_head->block_num()));
+               }
```

Comment:
Didn't actually mean to commit this, but not sure it is worth creating a diff PR for it.
It reduces log spam while syncing.

Reply:
It's OK to come along with this PR.